plugins/v010-adapter/0.1.0/README.md

```diff
@@ -0,0 +1,6 @@
+# Node Resource Interface v0.1.0
+
+*This version of NRI is supported through the included v010-adapter plugin.*
+
+Refer to the original documentation: [NRI v0.1.0](https://github.com/containerd/nri/blob/v0.9.0/README-v0.1.0.md).
+If you're still using *nri v0.1.0*, We recommend refactoring to the latest NRI.
```

Comment:
bring in the background v.0.1.0 paragraph from the readme.md and add links to the now removed v010 example/skel

.... other 0.1.0 files useful for v0.1.0 cll invoke style NRI plugins..
[v0.1.0 clearcfs example](https://github.com/containerd/nri/blob/v0.9.0/examples)
[v0.1.0 skel example](https://github.com/containerd/nri/blob/v0.9.0/skel)